tcf_website/templates/reviews/new_review.html

```diff
@@ -34,6 +35,8 @@ <h1 class="h2 mt-2 mb-4">Write a review</h1>
     $('[data-toggle="tooltip"]').tooltip()
   });
 
+  // Counter to keep track of number of times "zero-hours-review" modal is shown
+  var count = 0;
```

Comment:
Since the only values `count` ever takes on are `0` and `1`, it might be better to make it a boolean?

Reply:
hmmm yeah good call, would be more intuitive